### ISSUE TITLE
UnusedImport: misses import reference within GString

### ISSUE THREAD

**Issue by chrismair** [CodeNarc/CodeNarc#147]
UnusedImport: misses import reference within GString.

e.g. 
```
                import test.TestData

                void doStuff() {
                    final XML = """
                        <Request>
                            $TestData.GOOD_XML
                        </Request>
                        """
                    process(XML)
                }
```

**Comment by chrismair** [comment]
As a workaround, to avoid the rule violation, you can enclose the expression within the GString with braces, e.g.
`"$TestData.XML"`
becomes:
`"${TestData.XML}"`

**Comment by chrismair** [comment]
Available in CodeNarc 0.25.1
